install_for_analyst.bat

```diff
@@ -7,4 +7,4 @@
 
 set "MIRA_CMD_RUNNER_DIR=%ProgramFiles%\Mira Geoscience\Geoscience ANALYST\CmdRunner"
 set curdir=%~dp0
-"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --install "%curdir%\environments\conda-py-3.10-win-64.lock.yml" --with-pip-deps %*
+"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --install "%curdir%\environments\conda-py-3.10-win-64.lock.yml" --with-pip-deps %* --run pip install -e .[dash] %*
```

Comment:
I had hacked a quick thing  to install petro-lingo all from pip under a new env for Analyst,
but that does not quite apply here.

Command is more:
```suggestion
"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --name peak-finder --install "%curdir%environments\py-3.10-win-64.conda.lock.yml" ^
  --run "pip install -e .[dash]" %*
```

Reply:
update app in the first line comment:
```bat
:: Creates an environment for peak-finder-app under the Python environment location used by Geoscience Analyst.
```